crates/uv-python/fetch-download-metadata.py

```diff
@@ -1,272 +1,391 @@
-        if not rows:
-            break
-        for row in rows:
-            for asset in row["assets"]:
-                url = asset["browser_download_url"]
-                base_name = unquote(url.rsplit("/")[-1])
-                if base_name.endswith(".sha256"):
-                    continue
-                info = parse_filename(base_name)
-                if info is None:
-                    continue
-                py_ver, triple, flavor = info
-                if "-static" in triple or (flavor and "noopt" in flavor):
-                    continue
-                triple = normalize_triple(triple)
-                if triple is None:
-                    logging.debug("Skipping %s: unsupported triple", url)
-                    continue
-                results.setdefault(py_ver, []).append((triple, flavor, url))
-
-    # Collapse CPython variants to a single URL flavor per triple
-    cpython_results: dict[tuple[int, int, int], dict[tuple[str, str, str], str]] = {}
-    for py_ver, choices in results.items():
-        urls = {}
-        for triple, flavor, url in choices:
-            triple = tuple(triple.split("-"))
-            priority = _get_flavor_priority(flavor)
-            existing = urls.get(triple)
-            if existing:
-                _, _, existing_priority = existing
-                # Skip if we have a flavor with higher priority already
-                if priority >= existing_priority:
-                    continue
-            urls[triple] = (url, flavor, priority)
-
-        # Drop the priorities
-        cpython_results[tuple(map(int, py_ver.split(".")))] = {
-            triple: (url, flavor) for triple, (url, flavor, _) in urls.items()
-        }
+class CPythonFinder(Finder):
+    implementation = ImplementationName.CPYTHON
+
+    RELEASE_URL = (
+        "https://api.github.com/repos/indygreg/python-build-standalone/releases"
+    )
 
-    # Collect variants across interpreter kinds
-    # TODO(zanieb): Note we only support CPython downloads at this time
-    #               but this will include PyPy chain in the future.
-    final_results = {}
-    for interpreter, py_ver, choices in sorted(
-        chain(
-            (("cpython",) + x for x in cpython_results.items()),
-        ),
-        key=_sort_by_interpreter_and_version,
-        # Reverse the ordering so newer versions are first
-        reverse=True,
-    ):
-        # Sort by the remaining information for determinism
-        # This groups download metadata in triple component order
-        for (arch, operating_system, libc), (url, flavor) in sorted(choices.items()):
-            key = "%s-%s.%s.%s-%s-%s-%s" % (
-                interpreter,
-                *py_ver,
-                operating_system,
-                arch,
-                libc,
+    FLAVOR_PREFERENCES = [
+        "install_only",
+        "shared-pgo",
+        "shared-noopt",
+        "static-noopt",
+        "pgo+lto",
+        "pgo",
+        "lto",
+        "debug",
```

Comment:
Our current metadata includes `debug` flavors actually. I need to move it from `HIDDEN_FLAVORS` to keep the `download-metadata.json` file unchanged.